crates/iceberg/src/spec/schema.rs

```diff
@@ -32,16 +32,18 @@ use std::sync::Arc;
 
 use _serde::SchemaEnum;
 
+/// Type alias for schema id.
+pub type SchemaId = i32;
```

Comment:
I add this because I found that in different places we used different type, e.g. `i32`, `i64`. I think it improved readability, while without introducing any extra cost.